### DISCUSSION TITLE
Adding personal keybindings?

### DISCUSSION THREAD

**Discussion by amca01** [James-Yu/LaTeX-Workshop#3425] [Q&A] [answered]
I'm trying to move (again!) to VS Code + LaTeX Workshop after about a million years using Emacs + AucTeX.   My trouble is that those million years have ingrained keybindings to the extent they have become totally automatic.  So what I'd like is to be able to add some of the standard AucTeX keybindings to LaTeX Workshop.   For example, to  add a new environment in Emacs + AucTeX, I use the keybinding C-c C-e.  Now I know that LaTeX Workshop provides even easier ways of inserting a new environment, with Intellisense: \envname for example.  But C-c C-e is so automatic for me that my fingers will enter it almost as soon as I think about environments.  Is there a method by which I can add some of my most ingrained keybindings into LaTeX workshop?

**Comment by James-Yu** [comment, marked as answer]
Custom keybinding is to be provided by vscode instead of the extension. You may find vscode's doc on their website useful.

**Reply by amca01** [reply]
Thank you very much!  Well, as I'm sort of new to VS Code (I tried it a year ago) I wasn't quite sure of the best approach.  As always, RTFM is a good start ...
